exporter/splunkhecexporter/README.md

```diff
@@ -115,6 +119,14 @@ exporters:
     splunk_app_name: "OpenTelemetry-Collector Splunk Exporter"
     # Application version is used to track telemetry information for Splunk App's using HEC by App version.
     splunk_app_version: "v0.0.1"
+    hec_heartbeat:
+      interval: 30s
+    telemetry:
+      enabled: true
+      metrics_prefix: custom_app_metrics_prefix
+      extra_attributes:
+        dataset_name: SplunkCloudBeaverStack
+        custom_key: custom_value
```

Comment:
Do you need these attributes applied to HEC exporter metrics only? If it's needed for all collector's metrics, you can use `service.telemetry.resource` https://github.com/open-telemetry/opentelemetry-collector/blob/main/service/telemetry/config.go#L35

Reply:
Our use case is that we can spawn multiple exporters with each one having its own `dataset_name`. so it isn't applied globally.